drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -134,8 +147,16 @@ matterGeneric:
   - id: "matter/color/temp/light"
     deviceLabel: Matter Color Temperature Light
     deviceTypes:
+      - id: 0x010C # Color Temperature Light
       - id: 0x010D # Extended Color Light
+    deviceProfileName: light-color-level
+  - id: "matter/color/temp/light/2"
+    deviceLabel: Matter Color Temperature Light
```

Comment:
nit: should this be `Matter Extended Color Light`? Same for the fingerprint above

Reply:
I went with "Matter Color Light" which seems more user friendly to me.